website/docs/docs/core/connect-data-platform/snowflake-setup.md

```diff
@@ -421,6 +421,30 @@ my-snowflake-db:
 
 Refer to the [Snowflake docs](https://docs.snowflake.com/en/sql-reference/parameters.html#label-allow-id-token) for info on how to enable this feature in your account.
 
+### OAuth authorization
+
+See the Snowflake [docs](https://docs.snowflake.com/en/user-guide/oauth-snowflake-overview) for configuring OAuth. Your Snowflake admin needs to generate an [OAuth token](https://community.snowflake.com/s/article/HOW-TO-OAUTH-TOKEN-GENERATION-USING-SNOWFLAKE-CUSTOM-OAUTH) for your configuration to work.
+
+The OAUTH_REDIRECT_URI you provide in Snowflake is `http://localhost:PORT_NUMBER`. For example, `http://localhost:8080`.
```

Comment:
```suggestion
Provide the OAUTH_REDIRECT_URI in Snowflake:`http://localhost:PORT_NUMBER`. For example, `http://localhost:8080`.
```

Reply:
They don't provide it in our product; it's in the Snowflake docs.